public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```

Comment:
![image](https://github.com/ometman/math-magicians-app/assets/93039359/db3fbee2-ee69-4314-948f-13de1a5165e9)

- [ ] It would be nice if you remove the highlighted files in the image above to follow the project requirements which was to remove all the boilerplate files created by CRA 👍

Reply:
Whoosh, I thought I did. Thanks for catching that.